WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -43,6 +43,7 @@
 static NSString* const INCLUDE_NON_MODAL_ELEMENTS = @"includeNonModalElements";
 static NSString* const ACCEPT_ALERT_BUTTON_SELECTOR = @"acceptAlertButtonSelector";
 static NSString* const DISMISS_ALERT_BUTTON_SELECTOR = @"dismissAlertButtonSelector";
+static NSString* const SKIP_ADJUSTING_SCREENSHOT_ORIENTATION = @"skipAdjustingScreenshotOrientation";
```

Comment:
how about `autoAdjustScreenshotOrientation` and set it to `true` by default?

Reply:
ah. it also looks good. what about 8d033b23?